.github/workflows/integration_tests.yml

```diff
@@ -269,15 +286,21 @@ jobs:
           source test/utils.sh
           echo "Verifying that services and namespaces have been created"
 
-          # verify the deployments within the keptn namespace
+          # verify the deployments within the keptn namespace (for keptn control plane)
```

Comment:
please also add the statistics-service to the checks

Reply:
done. I also rebased from master branch, because statistics-service was not part of this when I started.